src/cmdlib.sh

```diff
@@ -414,20 +414,36 @@ EOF
         # the same RPMs: the `dnf repoquery` below is to pick the latest one
         dnf repoquery  --repofrompath=tmp,"file://${overridesdir}/rpm" \
             --disablerepo '*' --enablerepo tmp --refresh --latest-limit 1 \
-            --exclude '*.src' --qf '%{NAME}\t%{EVR}\t%{ARCH}' --quiet | python3 -c '
+            --exclude '*.src' --qf '%{NAME}\t%{EVR}\t%{ARCH}' \
+            --quiet > "${tmp_overridesdir}/pkgs.txt"
+
+        < "${tmp_overridesdir}/pkgs.txt" python3 -c '
 import sys, json
 lockfile = {"packages": {}}
 for line in sys.stdin:
     name, evr, arch = line.strip().split("\t")
     lockfile["packages"][name] = {"evra": f"{evr}.{arch}"}
 json.dump(lockfile, sys.stdout)' > "${local_overrides_lockfile}"
 
+        < "${tmp_overridesdir}/pkgs.txt" python3 -c '
+import sys, yaml
+manifest = {
+    "repos": ["coreos-assembler-local-overrides"],
+    "repo-packages": [
+        {
+            "repo": "coreos-assembler-local-overrides",
+            "packages": []
+        }
+    ]
+}
+for line in sys.stdin:
```

Comment:
I know this isn't a new issue, but hopefully in the future we can drop this python-in-shell-script in favor of [Go](https://github.com/coreos/coreos-assembler/issues/2821)